activerecord/test/cases/test_fixtures_test.rb

```diff
@@ -17,4 +17,18 @@ def test_use_transactional_tests_can_be_overridden
 
     assert_equal "foobar", @klass.use_transactional_tests
   end
+
+  def test_doesnt_rely_on_active_support_test_case_specific_method
+    klass = Class.new(Minitest::Test)
+    klass.include(ActiveRecord::TestFixtures)
+
+    assert_nothing_raised do
+      klass.stub(:setup_fixture_accessors, nil) do
+        klass.fixture_path = "some_path"
+        klass.fixtures(:all)
+      end
+    end
+
+    assert(klass.new("fake").run_in_transaction?)
```

Comment:
This works, but could you could probably get more coverage from running an empty test. WDYT?

Reply:
Yep , done